controllers/records.js

```diff
@@ -22,13 +22,11 @@ var request = function(opts, callback) {
 var createByForm = function(data, callback) {
   var required = ['message', 'from'],
       optional = ['reported_date', 'locale', 'gateway_ref'];
-  for (var k in required) {
-    if (required.hasOwnProperty(k)) {
-      if (!exists(data[required[k]])) {
```

Comment:
Yeah this commit broke. It was picked up in tests and I fixed it in a later commit in this PR.